catalogs/catalog-hadoop/build.gradle.kts

```diff
@@ -52,6 +52,8 @@ dependencies {
     exclude("org.eclipse.jetty", "*")
     exclude("io.netty")
     exclude("org.fusesource.leveldbjni")
+    // Exclude protobuf-java to avoid conflict with;
+    exclude("com.google.protobuf", "protobuf-java")
```

Comment:
is it still useful as you exclude the package in copy task

Reply:
Is exclude the `protobuf-java` in copy task is more general solution? since other package may introduce `protobuf-java` too.

Reply:
I'm afraid excluding it in the gradle dependencies may be more general. 

> since other package may introduce protobuf-java too.

Some modules use copy task to exclude jars because using `excluding` in gradle does not work. I believe this is not a very elegant behaviour.